graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -58,12 +58,13 @@ def tempdir_fixture():
 
 @pytest.fixture(scope="function", name="data_configs_with_label")
 def data_configs_with_label_fixture():
-    """Create data configuration object that contain labels"""
-    config_path = os.path.join(_ROOT, "resources/small_heterogeneous_graph/gconstruct-config.json")
+    """Create data configuration object that contain features and labels"""
+    config_path = os.path.join(
+        _ROOT, "resources/small_heterogeneous_graph/gsprocessing-config.json"
```

Comment:
Can we use /tmp/ ?

Reply:
These input files are bundled with the repo, we use temp directories for the output.